spec/sidekiq_prometheus/job_metrics_spec.rb

```diff
@@ -9,6 +9,13 @@ def prometheus_labels
     end
   end
 
+  module Sidekiq
+    module Limiter
+      class OverLimit < StandardError
+      end
+    end
+  end
```

Comment:
more fixes for rubocop:

```suggestion
class Sidekiq::Limiter::OverLimit < StandardError; end
```